elements/lisk-dpos/src/delegates_list.ts

```diff
@@ -213,8 +213,11 @@ const _pickStandByDelegate = (
 ): number => {
 	const seedNumber = randomSeed.readBigUInt64BE();
 	const totalVoteWeight = _getTotalVoteWeight(delegateWeights);
+
+	// To pass voteWeight = 0, initialize threshold with negative value
 	// tslint:disable-next-line no-let
-	let threshold = seedNumber % totalVoteWeight;
+	let threshold =
+		totalVoteWeight === BigInt(0) ? BigInt(-1) : seedNumber % totalVoteWeight;
```

Comment:
This is not true because totalVoteWeight can be non zero, and we still pick top 103.
This function is specifically to select standby delegates with voteWeight, so if we pick top 103, it shouldn't come to this function

Reply:
Yes `totalVoteWeight` can be non-zero. This logic is not enforcing it to be zero. 

We pick 103 delegates for forging, but not `top` 103. This `top` adjective is only for `101` delegates which we call active delegates. And rest of the 2 which we call standby delegates will be picked randomly based on a random seed.